enlighten/BusinessObjects.py

```diff
@@ -1,3 +1,4 @@
+from asyncio.windows_events import CONNECT_PIPE_INIT_DELAY
```

Comment:
Not sure why this was added. Maybe a weird linter autoimport. This should be removed though. It prevents the linux version from running since it is windows specific